playbooks/prepare_stack.yaml

```diff
@@ -97,7 +97,8 @@
       fi
       if ! openstack subnet show hostonly-subnet; then
           openstack subnet create --project openshift hostonly-subnet --subnet-range "{{ hostonly_cidr }}" \
-              --no-dhcp --gateway "{{ hostonly_gateway }}" \
+              --dhcp --gateway "{{ hostonly_gateway }}" \
+              --dns-nameserver "{{ network_info.dns | first }}" \
```

Comment:
```
  --dns-nameserver <dns-nameserver>
                        DNS server for this subnet (repeat option to set multiple DNS servers)
```

Is it worth repeating this for all nameservers?

Reply:
Depends how much effort it is. Nice to have, hence I was happy to merge without it.